test/unit/partitioner_gpt_test.py

```diff
@@ -57,3 +54,11 @@ def test_set_flag(self, mock_command):
     def test_set_flag_ignored(self, mock_warn):
         self.partitioner.set_flag(1, 'f.active')
         assert mock_warn.called
+
+    @patch('kiwi.partitioner.gpt.Command.run')
+    def test_set_hybrid_mbr(self, mock_command):
+        self.partitioner.partition_id = 3
+        self.partitioner.set_hybrid_mbr()
+        mock_command.assert_called_once_with(
+            ['sgdisk', '-h', '1:2:3', '/dev/loop0']
```

Comment:
ok, I'll merge it then. There will be a follow up request for the boot code to keep the hybrid setup after repartition. Would be great if you can have a look at that one too. I'll ping you once I'm done